tests/assets/packages/demo_package.py

```diff
@@ -48,6 +48,14 @@ def resp_dict(self) -> InvocableResponse[dict]:
     def resp_obj(self) -> InvocableResponse[dict]:
         return InvocableResponse(json=TestObj(name="Foo"))
 
+    @post("resp_true")
+    def resp_true(self) -> InvocableResponse[dict]:
+        return InvocableResponse(json=True)
+
+    @post("resp_false")
+    def resp_false(self) -> InvocableResponse[dict]:
+        return InvocableResponse(json=False)
+
```

Comment:
Was this commit intended for this PR?

Reply:
Yeah I added it while trying to debug, and then left it because `false` tends to be a really hard thing to represent in a multi-platform system.. so as silly as it sounds, it feels very comforting to know we have a test that tests for our ability to "return false" across Python -> Swift -> HTTP